pkg/api/validation/validation_test.go

```diff
@@ -1552,6 +1650,75 @@ func TestValidateVolumes(t *testing.T) {
 			},
 		},
 		{
+			name: "downapi valid defaultMode",
```

Comment:
@pmorie should I keep it like this, then? It is used like this in all those tests IIRC (those tests were re-written)